ivy/functional/ivy/general.py

```diff
@@ -2050,7 +2050,21 @@ def get_all_arrays_in_memory():
 
 @handle_exceptions
 def num_arrays_in_memory():
-    """Returns the number of arrays which are currently alive."""
+    """Returns the number of arrays which are currently alive.
```

Comment:
@tamukagate thanks for your contribution! I can see that the docstring tests are failing for the changes you made. I'll merge the PR once they are fixed :)

Reply:
Hi @rahulsnkr. I'm currently stuck on the implementation of the docstring examples. Please may you help me

Reply:
Hi @rahulsnkr, I've reverted the commits and rerun the tests